enhancements/api-review/azure_user_defined_tags.md

```diff
@@ -0,0 +1,322 @@
+                  azure:
+                    properties:
+                      resourceTags:
+                        description: resourceTags is a list of additional tags to apply to Azure
+                        resources created for the cluster. See 
+                        https://docs.microsoft.com/en-us/rest/api/resources/tags for information 
+                        on tagging Azure resources. Azure supports a maximum of 50 tags per 
+                        resource except for few, which have limitation of 15 tags. OpenShift 
+                        reserves 10 tags for its internal use, and allows 5 tags 
+                        for user configuration.
+                        type: array
+                        maxItems: 5
+                        items:
+                          description: AzureResourceTag is a tag to apply to Azure resources created for the cluster.
+                          type: object
+                          required:
+                            - key
+                            - value
+                          properties:
+                            key:
+                              description: key is the key of the tag
+                              type: string
+                              maxLength: 128
+                              minLength: 1
+                              pattern: ^[a-zA-Z][0-9A-Za-z_.=+-@]+$
+                            value:
+                              description: value is the value of the tag.
+                              type: string
+                              maxLength: 256
+                              minLength: 1
+                              pattern: ^[0-9A-Za-z_.=+-@]+$
+```
+
+### Implementation Details/Notes/Constraints [optional]
+Add a new field `resourceTags` to `.status.platformStatus.azure` of the 
+`infrastructure.config.openshift.io` type. Tags included in the `resourceTags` field 
+will be applied to new resources created for the cluster by the in-cluster operators.
+
+The `resourceTags` field in `status.platformStatus.azure` of `infrastructure.config.openshift.io`
+will be populated by the installer using the entries from `platform.azure.userTags` field of `install-config`.
+
+`status.platformStatus.azure` of `infrastructure.config.openshift.io` is immutable.
+
+All operators that create Azure resources will apply these tags to all Azure 
+resources they create.
+
+| Operator | Resources created by the operator |
+| -------- | ----------------------------- |
+| cloud-network-config-controller | Private IP address |
+| cluster-image-registry-operator | Storage Account |
+| cluster-ingress-operator | Load Balancer, DNS records |
+| cloud-credential-operator | IAM roles and policies |
+| machine-api-provider-azure | Application Security Group, Availability Set, Group, Load Balancer, Public IP Address, Route, Network Security Group, Virtual Machine Extension, Virtual Interface, Virtual Machine, Virtual Network. |
```

Comment:
is this the same set of resource owning components as for AWS?  Or is there a delta?

Reply:
AWS has a few of more operators(ex: aws-ebs-csi-driver-operator, aws-ebs-csi-driver-operator), but for Azure there is no new operator which wasn't there for AWS.

Reply:
you might mention that (it reduces the risk/complexity of adding this feature since it's all areas we've touched before at some level)

Reply:
comment incorporated.